src/Transition/InputRulesFiles/Rules8-7-0-to-8-8-0.md

```diff
@@ -8,17 +8,22 @@ This was previously an Excel workbook that made for very difficult version contr
 
 Insert new blank field 5 (A5) (for System Inlet Air Node Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Table:TwoIndependentVariables`
 
 Insert new blank field 14 (A7) (for External File Name). 
 Shift all later fields down by one.
-=======
 
 # Object Change: `Output:Surfaces:List`
 
 The only change is for field F1, which is A1.  Logic to apply:
 
 if key = 'DecayCurvesfromZoneComponentLoads', change to = 'DecayCurvesFromComponentLoadsSummary'
 
+# Object Change: `UnitarySystemPerformance:Multispeed`
+
+Fields 1-4 remain the same.  
+After F4, insert one new blank field that represents the No Load Supply Air Flow Rate Ratio. 
+Shift all later fields down by one.
```

Comment:
I have had to merge the rules file several times. I can't say that I paid close attention to what was being merged as how it needed to be merged. I am a little surprised this snuck in. It might be because I saved a branch and then popped it back in the wrong branch. I've done this several times lately and have been switching branches a lot to try to keep up. That's my only guess.